README.md

```diff
@@ -55,6 +63,64 @@ validate('invalid_dl');
 // => null
 ```
 
+## validateForState
+
+```js
+import { validateForState } from 'driver-license-validator';
+
+validateForState('A1234567', 'CO');
+// => false
+
+validateForState('A1234567', 'NY');
+// => true
+
+validateForState('invalid_dl');
+// => false
+```
+
+## requirementsByState
+
+```js
+import { requirementsByState } from 'driver-license-validator';
+
+requirementsByState('CO');
```

Comment:
What's the use case for getting the underlying format objects? I'm curious if maybe it would be better/easier to simply export the formats so they can be public.

```
import { US_DL }  from 'driver-license-validator';

console.log(US_DL["CO"]);
```

Reply:
Yes, this would be fine as well.